doc/source/whatsnew/v2.0.0.rst

```diff
@@ -596,6 +596,11 @@ Other API changes
   or :attr:`~DataFrame.iloc` (thus, ``df.loc[:, :]`` or ``df.iloc[:, :]``) now returns a
   new DataFrame (shallow copy) instead of the original DataFrame, consistent with other
   methods to get a full slice (for example ``df.loc[:]`` or ``df[:]``) (:issue:`49469`)
+- The :class:`Series` and :class:`DataFrame` constructors will now return a shallow copy
+  (i.e. share data, but not attributes) when passed a Series and DataFrame, respectively,
+  and with the default of ``copy=False`` (and if no other triggers a copy). Previously,
```

Comment:
"and if no other triggers a copy" is there a word missing here?

Reply:
Yes, I suppose I meant  "no other _keyword_ triggers a copy", eg by passing a dtype, or by passing index/columns causing a reindex.